autorest/codegen/templates/config.py.jinja2

```diff
@@ -61,8 +61,8 @@ class {{ code_model.class_name }}Configuration(Configuration):
 {% if code_model.options['credential_scopes'] %}
         self.credential_scopes = {{ code_model.options['credential_scopes'] }}
 {% endif %}
+        kwargs.setdefault('sdk_moniker', '{{ code_model.class_name|lower }}/{}'.format(VERSION))
```

Comment:
Should be the package-name, and only class_name if no "package-name" was provided